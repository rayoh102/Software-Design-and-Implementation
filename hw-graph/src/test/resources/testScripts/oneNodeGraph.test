# Tests if a node can be added to the graph

# Create a graph
CreateGraph oneNodeGraph

# Add node
AddNode oneNodeGraph a

# Print all the nodes in the graph and the outgoing edges from a
ListNodes oneNodeGraph
ListChildren oneNodeGraph a

