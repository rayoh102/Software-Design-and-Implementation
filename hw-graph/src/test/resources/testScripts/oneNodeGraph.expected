# Tests if a node can be added to the graph

# Create a graph
created graph oneNodeGraph

# Add node
added node a to oneNodeGraph


# Print all the nodes in the graph and the outgoing edges from a
oneNodeGraph contains: a
the children of a in oneNodeGraph are: